src/mugs.js

```diff
@@ -1,3 +1,37 @@
+/**
+ * A mug is a question, containing data, bind, and control elements.
+ *
+ * Main Properties
+ *  nodeID
+ *  label
```

Comment:
I wonder how many forms exist that use `label` and `hintLabel`? (and actually need them)

I can't imagine a future where we're not using javaRosa and would want the code that implements `label` and `hintLabel`. I think the main reasons for keeping them is theoretical backward compatibility and plugin system purity.